digiwf-libs/digiwf-integration-e2e-test/digiwf-integration-e2e-test-core/src/main/java/de/muenchen/oss/digiwf/integration/e2e/test/DigiWFIntegrationE2eTestUtility.java

```diff
@@ -0,0 +1,49 @@
+package de.muenchen.oss.digiwf.integration.e2e.test;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import de.muenchen.oss.digiwf.message.core.api.MessageApi;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import static de.muenchen.oss.digiwf.message.common.MessageConstants.*;
+import static org.awaitility.Awaitility.await;
+
+@Component
+@RequiredArgsConstructor
+public class DigiWFIntegrationE2eTestUtility {
+
+    private final MessageApi messageApi;
+    private final TestMessageConsumer testMessageConsumer;
+    @Value("${spring.cloud.stream.bindings.functionRouter-in-0.destination}")
+    private String messageTopic;
+
+
+    public void setupWiremock(final String url, final String expectedResponse) {
```

Comment:
Könnten wir das ggf in eine extra Komponente auslagern? da kommen ggf. noch einige andere Funktionen für Wiremock dazu (bei OKewo bspw. schon 2 andere).